src/clib/cie_thin_cooling_rate_tables.h

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************/
+/*
+ * ---------Arrays of data used within cie_thin_cooling_rate_g.c---------
+ * 
+ * These arrays contain data neccessary for calculating the CIE optically thin
+ * cooling rate. The data here is taken from the original Grackle fortran
+ * subroutine which calculated the CIE rate.
+ * 
```

Comment:
Maybe useful to include the original comments here from the fortran version which give a little information about the provenance of these rates (Ripamonti & Abel 2003 I think).

Reply:
Yes this is a good point, those comments seem to have been lost -- will copy them across to where the cie_thin_cooling rate is calculated.